app/models/form.rb

```diff
@@ -3,18 +3,18 @@ class Form
               :start_date, :end_date, :type, :name, :setup_definition,
               :setup_sections, :form_sections
 
-  def initialize(form_path, name, sections_in_form = [], type = "lettings")
+  def initialize(form_path, start_year = "", sections_in_form = [], type = "lettings")
     if type == "sales"
-      @name = name
+      @name = "#{start_year}_#{start_year + 1}_sales"
```

Comment:
```suggestion
      @name = "#{start_year}_#{start_year + 1}_#{type}"
```

You can probably shift this out if the `if` condition here because it should be shared between types.